demo/src/test/java/com/mongodb/jdbc/utils/TestUtils.java

```diff
@@ -0,0 +1,439 @@
+package com.mongodb.jdbc.utils;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public abstract class TestUtils {
```

Comment:
I'm unsure the purpose of this file. The `PrintUtils` class alone is very helpful going forward as it gives building blocks for anyone to easily print result sets and metadata during development/testing. This file, however, has unclear utility. The "test" methods aren't actually testing anything -- they are just printing data  for various `DatabaseMetaData` methods.

Whenever there is a specific issue someone wants to investigate, they'll need to alter the "tests" here so they could print what they need to print. Are these meant to just be reference methods that people alter as needed? What is the expected future use of this file and the test methods in it?